src/Kunstmaan/SeoBundle/Event/RobotsEvent.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Kunstmaan\SeoBundle\Event;
+
+class RobotsEvent
```

Comment:
final

Reply:
Deze ging niet, want anders kan ik hem niet gebruiken in phpspec als collaborator

Reply:
Ik verwacht dat dit vanuit Kunstmaan wel moet. Dit was de vorige keer ook op een andere class.